dev_setup.sh

```diff
@@ -14,23 +14,25 @@ set -e
 
 HERE=$(cd `dirname $0`; pwd)
 LOG=$HERE/dev_setup.log
+ROOT=$HERE
+ENV=$ROOT/env.sh
```

Comment:
Don't call this ```ENV```; that means something magic to the shell (it's a misfeature for doing something similar to .profile)

:-(

Reply:
oooops